book/languages/csharp/advantages.md

```diff
@@ -1,14 +1,14 @@
 # Advantages
 
-C# was designed to be simple and easy to use, it's syntax is very natural and non-verbose. As a high-level language, it abstracts away most of the complex details, giving the user an ability to focus on its pure programming tasks instead of worrying about the little bits.
+C# was designed to be simple and easy to use, its syntax is very natural and non-verbose. As a high-level language, it abstracts away most of the complex details, giving the user an ability to focus on its pure programming tasks instead of worrying about the little bits.
 
-While abstracting away the details, C# doesn't limit your abilities and gives you the option to work on a lower level using features like Interoperability which enables you to take advantage of unmanaged code such as C/C++ directly from your code.
+While abstracting away the details, C# doesn't limit your abilities and gives you the option to work on a lower level using features like *Interoperability* which enables you to take advantage of unmanaged code such as C/C++ directly from your code.
 
-C# is a managed programming language. Being one, allows you to not worry about freeing up the memory. The memory cleanup is dealt by the GC (Garbage collector) of the .NET Framework.
+C# is a managed programming language which means the programmer need not worry about memory management. It is dealt by the GC (Garbage collector) of the .NET Framework.
 
 Another advantage of managed code is that it becomes platform-independent, meaning that it can be run on any platform with .Net Framework installed, but it's not limited to it.
-Thanks to Mono project, there are other technologies that support C# as it's primary language. Tools like Unity and Xamarin allow you to write C# code and run it on any platform of your choice.
+Thanks to Mono project, there are other technologies that support C# as its primary language. Tools like Unity and Xamarin allow you to write C# code and run it on any platform of your choice.
 
-It's a type-safe compiled language, meaning that your code is verified by the compiler before it's execution, making your code more resilient to runtime errors by catching them ahead of time.
+It's a type-safe compiled language, meaning that your code is verified by the compiler before it111s execution, making your code more resilient to runtime errors by catching them ahead of time.
```

Comment:
typo: before **it111s** execution

Reply:
??

Reply:
I'll clarify. In your commit: Typos and slight modifications. https://github.com/devRant-Community/Community-Programming-Book/pull/31/commits/0fcb1e82f173a166ff42351a5a42c96b0acc5bd3
You did a typo: it111s  instead of it's